packages/@lwc/template-compiler/typings/parse5-internals.d.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
`parse5` v6 is not written in TypeScript. My initial naive implementation upgraded `parse5` to v7, which is implemented in TypeScript and exports a lot of the internal types. This made the explicit type definition here unnecessary.

However, it also introduced unrelated changes into the codebase. In order to keep things clean, I've left `parse5` at v6. We can revisit in a follow-up PR or at a later date, if desired.

Reply:
Can we add a TODO to this file to remove it when we can? I think that's what we typically do when we have to embed our own types.